test/unit/jest.config.json

```diff
@@ -25,5 +25,8 @@
 		"/test/e2e",
 		"<rootDir>/.*/build/",
 		"<rootDir>/.*/build-module/"
+	],
+	"modulePathIgnorePatterns": [
+		"<rootDir>/gutenberg-mobile/"
```

Comment:
Should we include this for `coveragePathIgnorePatterns` ?

Reply:
Oh, good point, seems like we should. Will look into it now.

Reply:
Done with 24cd903.